test/modules/TestResultCoordinator/Reports/TwinCountingReport.cs

```diff
@@ -32,7 +32,19 @@ public TwinCountingReport(string testDescription, string trackingId, string expe
 
         public ReadOnlyCollection<string> UnmatchedResults { get; }
 
-        public override bool IsPassed => this.TotalExpectCount == this.TotalMatchCount && this.TotalExpectCount > 0;
+        public override bool IsPassed => this.IsPassedHelper();
+
+        bool IsPassedHelper()
+        {
+            if (this.TestDescription.Contains("desired property"))
```

Comment:
Can you make add a description to the code as to what specific tests is this pointing too